dojo/tools/qualys/parser.py

```diff
@@ -252,6 +255,8 @@ def parse_finding(host, tree):
         finding.active = _temp["active"]
         if _temp.get("CVSS_vector") is not None:
             finding.cvssv3 = _temp.get("CVSS_vector")
+        if _temp.get("CVSS_value") is not None:
+            finding.cvssv3_score = _temp.get("CVSS_value")
```

Comment:
could you please add a unittest for that?

Reply:
Alright I added a unit-test for the case that a Finding has only a cvss_score in the detections, has a score and a vector in the detections and if it has no score and no vector in the detections and not in the knowledgebase.